docs/guides/qiskit-addons-cutting.mdx

```diff
@@ -0,0 +1,154 @@
+
+### A short example: cutting a RZZGate
+
+As a basic explicit example, consider the decomposition of a cut RZZGate (details can be found in [[2]](#references)). A quantum circuit that contains an RZZGate can be simulated by performing six subexperiments where the RZZGate has been replaced with only single-qubit operations (these are the $\mathcal{F}_i$'s from the equation above). The results of this circuit are reconstructed by combining the results of each subexperiment alongside a set of coefficients (the $a_i$'s from the equation above), which can be either positive or negative.
+
+For some chosen $\theta$ parameter for the RZZGate, the six subexperiments are as follows:
+1. With coefficient $a_1 = \cos^2(\theta/2)$, do nothing ($I\otimes I$)
+1. With coefficient $a_2 = \sin^2(\theta/2)$, perform a ZGate on each qubit ($Z\otimes Z$)
+1. With coefficient $a_3 = -\sin(\theta)/2$, perform a projective measurement in the $Z$ basis on the first qubit and an $S$ on the second ($M_z\otimes S^\dagger$). If the result of the measurement is $1$, flip the sign of that outcome's contribution during reconstruction.
+1. With coefficient $a_4 = \sin(\theta)/2$, perform a projective measurement in the $Z$ basis on the first qubit and an $S^\dagger$ on the second ($M_z\otimes S^\dagger$). If the result of the measurement is 1, flip the sign of that outcome's contribution during reconstruction.
+1. Same as 3. ($a_5=a_3$), but swap the qubits (perform $S\otimes M_z$ instead).
+1. Same as 4. ($a_6=a_4$), but swap the qubits (perform $S^\dagger\otimes M_z$ instead).
+
+### Sampling overhead reference table
+
+The following table provides the sampling overhead factor for a variety of two-qubit instructions, provided that only a single instruction is cut.
+| Instructions | KAK decomposition angles | Sampling overhead factor |
+| --- | --- | --- |
+| CSGate, CSdgGate, CSXGate | $\left(\pi/8, 0, 0\right)$ | $3+2\sqrt(2) \approx 2.828$ |
+| CXGate, CYGate, CZGate, GHGate, ECRGate | $\left(\pi/4, 0, 0\right)$ | $3^2=9$ |
+| iSwapGate, DCXGate | $\left(\pi/4, \pi/4, 0\right)$ | $7^2 = 49$ |
+| SwapGate | $\left(\pi/4, \pi/4, \pi/4\right)$ | $7^2 = 49$ |
+| RXXGate, RYYGate, RZZGate, RZXGate | $\left(\lvert\theta/2\rvert, 0, 0, \right)$ | $\left(1 + 2\lvert\sin(\theta)\rvert\right)^2$ |
+| CRXGate, CRYGate, CRZGate, CPhaseGate | $\left(\lvert\theta/4\rvert, 0, 0\right)$ |  $\left(1 + 2\lvert\sin(\theta/2)\rvert\right)^2$ |
+| XXPlusYYGate, XXMinusYYGate | $\left(\vert\theta/4\rvert, \lvert\theta/4\rvert, 0\right)$ | $\left(1 + 4\lvert\sin(\theta/2)\rvert + 2\sin^2(\theta/2)\right)^2$ (independent of $\beta$) |
+| Move (cut wire in the LO scenario) | N/A | $4^2 = 16$ |
+
+## Next steps
+
+<Admonition type="tip" title="Recommendations"> 
+    - [Get started with circuit cutting using wire cuts.](/guides/qiskit-addons-cutting-wires)
+    - [Get started with circuit cutting using gate cuts.](/guides/qiskit-addons-cutting-gates)
```

Comment:
Invert the ordering of these two next steps (to match the order in the left nav panel)